src/apiCall.js

```diff
@@ -1,7 +1,11 @@
+//import { useQuery, useMutation } from '@apollo/client'
+
 const fetchItems = async (tripSelection) => {
   const response = await fetch('https://packmate-be-d3fb267b5fee.herokuapp.com/graphql', {
     method: 'POST',
-    headers: { 'Content-Type': 'application/json' },
+
+    headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Credentials': 'true' },
```

Comment:
I tried this version of the header yesterday but i didn't have the correct dependencies downloaded.